spec/lib/initializer_spec.rb

```diff
@@ -0,0 +1,91 @@
+# require File.expand_path('../../lib/straight-server/initializer', File.dirname(__FILE__))
+require 'fileutils'
+require_relative '../../lib/straight-server/random_string'
+require_relative '../../lib/straight-server/initializer'
+require_relative '../../lib/straight-server/config'
+require_relative '../../lib/straight-server'
+
+RSpec.describe StraightServer::Initializer do
+
+  class StraightServer::TestInitializerClass
+    include StraightServer::Initializer
+    include StraightServer::Initializer::ConfigDir
+  end
+
+  before(:each) do
+    remove_temp_dir
+    templates_dir = File.expand_path('../../templates', File.dirname(__FILE__))
+    ENV['HOME']   = File.expand_path('../temp', File.dirname(__FILE__))
+    @test_class_object = StraightServer::TestInitializerClass.new
+    StraightServer::Initializer::ConfigDir.set!
+  end
+
+  after(:each) do
+    remove_temp_dir
+  end
+
+  
+  it "creates config files" do
+    begin
+      @test_class_object.create_config_files
+    rescue Exception => e
+      expect(e.status).to eq 0 
```

Comment:
Thats the main thing I wanted to ask. It looks ugly but specs are failing woth SystemExit: exit